HW2_Borovikova/calculator.py

```diff
@@ -0,0 +1,30 @@
+#  Addition
+def addition(n1, n2):
+	return n1 + n2
+
+#Divison
+def divison(x, y):
+	return x / y
+
+# Функция для вычитания 
+def subtract(a, b):
+	 return a - b 
+
+# Функция для умножения
+def multiply(a, b): 
+	return a * b 
+
+#Напишите функцию здесь
+def main():
+  built = (input()).split()
+  number1=float(built[0])
+  number2=float(built[2])
```

Comment:
принято ставить пробелы при декларации переменных  
```suggestion
  number1 = float(built[0])
  number2 = float(built[2])
```